client/lib/gsuite/index.js

```diff
@@ -30,3 +30,6 @@ export { hasGSuiteWithUs } from './has-gsuite-with-us';
 export { getGoogleMailServiceFamily } from './get-google-mail-service-family';
 export { getProductSlug, getProductType } from './gsuite-product-type';
 export { isPendingGSuiteTOSAcceptance } from './is-pending-gsuite-tos-acceptance';
+export { getGoogleExpiryDate } from './get-google-expiry-date';
+export { getGoogleMailboxPurchaseCost } from './get-google-mailbox-purchase-cost';
+export { getGoogleMailboxRenewalCost } from './get-google-mailbox-renewal.cost';
```

Comment:
I've made a similar comment in https://github.com/Automattic/wp-calypso/pull/61324 but I would suggest avoiding `Google` in the name of those selectors, as it's too generic. I would keep using `GSuite` since those selectors are not specific to a given product (which was not the case of `isDomainEligibleForGoogleWorkspaceFreeTrial()` in the other pull request) so it's consistent with existing selectors (that way we don't have to look up both `GSuite` and `GoogleWorkspace` when looking up a function).

Reply:
I saw that the folder contained a lot of "GSuite" prefixed/suffixed functions, but I also thought that we were going to rename all existing "GSuite" strings with "Google Workspace" gradually.